ddp-workspace/projects/ddp-pancan/src/app/components/splash-pages/colorectal-page/colorectal-page.component.scss

```diff
@@ -0,0 +1,212 @@
+            margin-top: 2.5rem;
+            margin-bottom: 2.5rem;
+        }
+    }
+
+    &.participate {
+        padding-top: 5.18rem;
+        padding-bottom: 3.75rem;
+        text-align: left;
+
+        ::ng-deep .container:last-child {
+            text-align: center;
+        }
+    }
+
+    &.faq {
+        padding-top: 6.875rem;
+        padding-bottom: 2.875rem;
+
+        .title {
+            margin-top: 0;
+        }
+
+        .container {
+            margin: 4rem auto 3.125rem;
+            max-width: 1007px;
+        }
+    }
+
+    &.partners {
+        max-width: 1081px;
+        margin: 0 auto;
+        padding-top: 4.875rem;
+        padding-bottom: 5.5rem;
+        text-align: left;
+
+        .title {
+            margin-top: 0;
+        }
+
+        .text {
+            opacity: 1;
+            margin-bottom: 0;
+        }
+
+        .logos {
+            display: flex;
+            gap: 5rem;
+            flex-wrap: wrap;
+            justify-content: space-between;
+        }
+    }
+
+    &.stay-informed {
+        padding-top: 5.375rem;
+        padding-bottom: 4.5rem;
+
+        ::ng-deep .container {
+            max-width: 1062px;
+        }
+    }
+
+    &.join-cmi {
+        padding-top: 4.75rem;
+        padding-bottom: 6.19rem;
+    }
+}
+
+.btn {
+    font-size: 1.13rem;
+    padding: 1rem 2rem;
+    line-height: 22px;
+}
+
+.join-cmi-btn {
+    font-size: 1.25rem;
+    margin: 1.56rem 0 7.5rem;
+}
+
+@media screen and (max-width: $desktop-breakpoint) {
+    section {
+        &.introduction {
+            padding-left: 3rem;
+            padding-right: 3rem;
+
+            .description {
+                min-width: 365px;
+            }
+
+            .image {
+                width: 320px;
+                height: 315px;
+            }
+        }
+    }
+}
+
+@media screen and (max-width: $tablet-breakpoint) {
+    section {
+        &.introduction {
+            .flex {
+                flex-direction: column;
+                align-items: center;
+                gap: 2rem;
+            }
+
+            .description {
+                text-align: center;
+                padding: 0;
+            }
+
+            .image {
+                width: 260px;
+                height: 255px;
+            }
+
+            .logos img:first-child {
+                margin-right: 3rem;
+            }
+        }
+
+        &.partners {
+            .logos {
+                flex-direction: column;
+            }
+        }
+    }
+}
+
+@media screen and (max-width: $mobile-breakpoint) {
+    section {
+        padding-left: 1.5rem !important;
+        padding-right: 1.5rem !important;
+
+        &.introduction {
+            .description {
+                min-width: unset;
+            }
+
+            .image {
+                display: none;
+            }
+
+            .logos img {
+                display: block;
+                margin: 0 auto 5px !important;
+            }
+        }
+    }
+}
```

Comment:
any chances we can move some of the styles out to styles folder and share with welcome page since the styles looks very close

Reply:
Refactored styles a bit.